### PR TITLE
Feature/playlistmodule

### PR DESCRIPTION
**Pull request by goody153**
## Ticket : 
 - [PlaylistModule](https://trello.com/c/XaSle5oq)

## Summary : 
 - Created Playlist Module
     - Added Models
     - Added Admin

### CODE REVIEW

File: playlist/models.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    """
+    playlist = models.ForeignKey(Playlist)
+    title = models.CharField(max_length=255)
+    link = models.URLField()
+    user = models.ForeignKey(User)
+    date_created = models.DateTimeField(auto_now_add=True)
+    date_updated = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return ("{}".format(self.title))
```
Comment (earvingemenez) [inline playlist/models.py]: what is the purpose of that open and close parenthesis wrapping in your return value

Comment (goody153) [reply playlist/models.py]: In this case it does nothing. Sorry sir going to remove it.

File: users/migrations/0002_user_updated_at.py [hunk 2]
```diff
@@ -1,20 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (earvingemenez) [inline users/migrations/0002_user_updated_at.py]: why did you remove this file?

**Comment by goody153** [comment]
Issues Fixed
